addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -1098,6 +1098,10 @@ var ListRenderer = BasicRenderer.extend({
     _onToggleOptionalColumn: function (ev) {
         var self = this;
         ev.stopPropagation();
+        // when the input's label is clicked, the click event is also raised on
+        // the input itself, so this handler is executed twice (except if the
+        // rendering is quick enough, as when we render, we empty the HTML)
+        ev.preventDefault();
```

Comment:
@aab-odoo 
When there is a piece of documentation that justifies a diff, I usually put the reference both in the code and in the commit message

Reply:
:+1: